Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.1.0"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LibGEOS = "a90b1aa1-3769-5649-ba7e-abc5a9d163eb"
+LibSpatialIndex = "f19c2e90-9d16-5f2d-a2a7-af3fb29e4907"
```

Comment:
Question: Did you have to go through any hoops to get LibSpatialIndex installed? Totally OK with using it—indeed, I believe I may have recommended it—but I've had trouble with it in the past in a Python context.

Reply:
Nope, not at all! Everything was pretty smooth for me.